### DISCUSSION TITLE
Contribution on profile section

### DISCUSSION THREAD

**Discussion by TobiasRuano** [community/community#3366] [Mobile] [answered]
Show more information regarding your contributions on the profile section. More like the website, were it shows the graph and all your contributions.

**Comment by mbp15** [comment]
> Maybe not on the iPhone, but it would be a shame not to implement this on iPads.

**Comment by candyho** [comment, marked as answer]
Hey, thanks for sharing your feedback! If you are on iOS we have some great news for you! We have just shipped the Contribution Graph Widget for the GitHub iOS app so that you can see your own progress on the go, right from the home screen of your iPhone or iPad. You can find more details here - https://github.blog/changelog/2021-02-02-github-for-ios-contribution-graph-widget/ 

Supporting contributions graph in-app isn't something on our roadmap yet though, but please keep the feedback coming, we will monitor the requests and re-evaluate priority from time to time!
